.github/workflows/deploy-apis-to-staging.yml

```diff
@@ -1802,8 +1802,5 @@ jobs:
           method: kubeconfig
           kubeconfig: ${{ secrets.K8S_CONFIG_STAGE }}
 
-      - name: Apply ingress changes
-        run: |
-          kubectl apply -f k8s/nginx/staging/ingress.yaml
-          kubectl apply -f k8s/nginx/staging/virtual-server.yaml
-          kubectl apply -f k8s/nginx/staging/global-config.yaml
+      - name: Apply nginx changes
+        run: kubectl apply -f k8s/nginx/staging/
```

Comment:
[update staging workflow to apply all files at once](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/c9c22af49604c96188a5f315f9931a99fa7770db)